runtime/src/main/java/com/dylibso/chicory/runtime/Module.java

```diff
@@ -76,63 +75,18 @@ public Logger logger() {
     }
 
     public Instance instantiate() {
-        return this.instantiate(new HostImports(), true);
+        return this.instantiate(new HostImports(), true, true);
     }
 
     public Instance instantiate(HostImports hostImports) {
-        return this.instantiate(hostImports, true);
+        return this.instantiate(hostImports, true, true);
     }
 
-    public Instance instantiate(HostImports hostImports, boolean start) {
+    public Instance instantiate(HostImports hostImports, boolean initialize, boolean start) {
         var globalInitializers = new Global[] {};
         if (this.module.globalSection() != null) {
             globalInitializers = this.module.globalSection().globals();
         }
-        var globals = new Value[globalInitializers.length];
```

Comment:
this is duplicated code, the logic should be executed during `Instance.initialize`